daemon/api_quotas.go

```diff
@@ -55,11 +56,12 @@ type postQuotaGroupData struct {
 }
 
 type quotaGroupResultJSON struct {
```

Comment:
as I commented in the other PR this could use the client struct directly nowadays

Reply:
yes I have it changed in a followup PR, I wanted to keep this as simple as and straight forward as possible to keep up progress